sphinx/source/demo.rst

```diff
@@ -30,7 +30,7 @@ Given the above ``scenario.json`` this should create 2 nodes on the local machin
     :end-before: SNIPPET_END: create_network
     :dedent: 2
 
-Each transaction listed in the scenario is then sent to either the `primary` or a `follower` node. As the current implementation uses Raft for consensus, any ``Write`` transactions (which modify the KV) must be sent to the `primary`. Sending them to a `follower` will result in a ``TX_NOT_LEADER`` error response. ``Read`` transactions can be go to any node.
+Each transaction listed in the scenario is then sent to either the `primary` or a `backup` node. As the current implementation uses Raft for consensus, any ``Write`` transactions (which modify the KV) must be sent to the `primary`. Sending them to a `backup` will result in a ``TX_NOT_PRIMARY`` error response. ``Read`` transactions can sent to any node.
```

Comment:
The last sentence here still doesn't flow. Also now that I'm reading this for the third time, I think the whole paragraph is out-of-date: We have forwarding now, so we (usually) don't return TX_NOT_PRIMARY? We should instead briefly describe the forwarding here - `Write` transactions are forwarded and only processed on the primary, `Read`s can be processed by any node (we should distinguish between the node where transactions may be _sent_ and where the associated work is actually done).

Reply:
Correct. `TX_NOT_PRIMARY` only really exists if, by the time a command is forwarded, the primary has changed (quite unlikely in practice).